### ISSUE TITLE
一系列google不能访问了

### ISSUE THREAD

**Issue by Usb233** [racaljk/hosts#181]
应该不止是我一个吧？

**Comment by stanzhou** [comment]
+1

**Comment by indexofire** [comment]
+1

**Comment by JHXue** [comment]
+1
+1

**Comment by K-Guan** [comment]
原来哦，我还以为是我的网络出了问题。我们会想办法解决的。

**Comment by K-Guan** [comment]
补充：Google全线服务皆挂。看来是IP `216.239.38.125` 又不行了：

    kevin@Arch ~> ping 216.239.38.125
    PING 216.239.38.125 (216.239.38.125) 56(84) bytes of data.
    ^C
    --- 216.239.38.125 ping statistics ---
    8 packets transmitted, 0 received, 100% packet loss, time 6999ms

**Comment by topcheer** [comment]
64.233.162.84 貌似还可以

**Comment by K-Guan** [comment]
@topcheer 确实唉，这个IP是？

**Comment by indexofire** [comment]
64.233.162.84 很慢

**Comment by K-Guan** [comment]
@topcheer  哦，原来是我们的Gmail hosts。刚刚测试了下，好像除了上面我提到的那个，挂掉的还不少呢。比如：

    173.194.69.95   ajax.googleapis.com

----------------

    kevin@Arch ~> ping 173.194.69.95
    PING 173.194.69.95 (173.194.69.95) 56(84) bytes of data.
    ^C
    --- 173.194.69.95 ping statistics ---
    10 packets transmitted, 0 received, 100% packet loss, time 9007ms

**Comment by K-Guan** [comment]
@indexofire Yep

**Comment by K-Guan** [comment]
@andytimes 现在Google的hosts极不稳定。比如说我刚刚收到了你的邮件，可是刚想回复连接就再次中断了。你那边还没问题？

**Comment by K-Guan** [comment]
@andytimes 现在Google的hosts极不稳定。比如说我刚刚收到了你的邮件，可是刚想回复连接就再次中断了。你那边还没问题？

**Comment by K-Guan** [comment]
@andytimes 现在Google的hosts极不稳定。比如说我刚刚收到了你的邮件，可是刚想回复连接就再次中断了。你那边还没问题？

**Comment by SamZhangGo** [comment]
me too +1

**Comment by andytimes** [comment]
@K-Guan 我一般都是用 SS ，我把我的 QQ 邮箱发给你，私聊。

**Comment by K-Guan** [comment]
@andytimes 然而现在Gmail不能用怎么私聊……话说我马上要睡了，明天再说好了。不过我正在研究Private Internet Access VPN，应该没问题咯。

**Comment by andytimes** [comment]
@K-Guan 我到你的个人仓库留下我的邮箱，用邮件交流

**Comment by charlestian** [comment]
得更新了

2016-01-26 23:36 GMT+08:00 Andy Deng <notifications@github.com>:

**Comment by aofall** [comment]
经路由追踪发现Google 主动屏蔽大陆的电信联通用户 :(
移动铁通教育网没事
问题出在谷歌骨干网 209.85.248.62

**Comment by hwk603** [comment]
+1【电信无法google

**Comment by 4679** [comment]
电信+1

**Comment by 1086** [comment]
+1

**Comment by rensss** [comment]
+1

**Comment by borisyu** [comment]
电信+1

**Comment by p0mp0k0** [comment]
电信+1

**Comment by dingjibang** [comment]
辛苦了，请加油！（/TДT)/

**Comment by Titansonia** [comment]
电信+1

**Comment by Eamonnzhang** [comment]
武汉联通+1

**Comment by X-man-rascal** [comment]
坐等更新

**Comment by K-Guan** [comment]
@aofall 嗯？是什么问题？

**Comment by mushanzi1216** [comment]
64.233.162.84 换成这个感觉速度还可以啊

**Comment by xiaoping378** [comment]
:+1: 
64.233.162.83 这个速度也行

**Comment by wolfan** [comment]
google.com 无法访问了，不过google.com.hk还可以，puls也不行了。

**Comment by allecs** [comment]
四川电信，64.233.162.83 还可以

**Comment by redapple0204** [comment]
+1 全部无法访问，只能先刮着vpn了。。。。

**Comment by Rodney2012** [comment]
+1.目前github的hosts地址还没有失效

**Comment by DuanTzXavier** [comment]
[刚刚找的,试过可以用](http://laod.cn/hosts/2016-google-hosts.html)

**Comment by omnbmh** [comment]
DuanTzXavier  可用 哇咔哇咔

**Comment by 1086** [comment]
64.233.162.83  浙江电信不行

**Comment by inix** [comment]
南宁联通，近一周无法使用谷歌了，请各位作者帮忙更新。android studio都无法下载sdk了。太郁闷了。
非常感谢

**Comment by nxtreaming** [comment]
如果不使用youtube等消耗带宽的服务，只是用Google其他服务，可以联系我，我免费提供Google代理给大家用。

**Comment by qiuyuntao** [comment]
@nxtreaming 怎么提供？我只求能google搜东西就可以了

**Comment by nxtreaming** [comment]
@qiuyuntao 我的github账号的profile中有邮件地址,给我发邮件,在邮件中注明你的github账号.

**Comment by aofall** [comment]
@K-Guan 不知道Google在搞什么。包括最近新开的4个IP段也是这样。
现在只能用GGC或者说那段很慢的gws替换，我不想公布GGC。你用64.233.162.8x 替换一下然后push吧

**Comment by nxtreaming** [comment]
用host的这种方案进行使用google的服务的兄弟应该大部分是程序员, 为什么不自己搭建一个翻墙服务呢,花不了多少钱, 重要的是这样稳定性和便利性要比host方式高很多,能节约大家的时间.

**Comment by 4679** [comment]
有ss 但是手机也不是一直开着ss 所以就用hosts咯

On Jan 27, 2016 22:59, Jack Waller <notifications@github.com> wrote:

用host的这种方案进行使用google的服务的兄弟应该大部分是程序员, 为什么不自己搭建一个翻墙服务呢,花不了多少钱, 重要的是这样稳定性和便利性要比host方式高很多,能节约大家的时间.

**Comment by K-Guan** [comment]
@aofall 这样呀。话说有多少要替换的？好像所有Google相关的域名都要替换吧？

**Comment by MakeForce** [comment]
@inix mirrors.neusoft.edu.cn 80  代理这个是东软信息学院的

**Comment by MakeForce** [comment]
@inix 下android SDK 还是可以的

**Comment by aofall** [comment]
@K-Guan 
216.239.38.125 ＞＞ 64.233.162.8x
216.239.38.21 ＞＞ 64.233.188.121 （这个你测试一下能不能Ping，我只发现到hk的线路出问题
173.194.69.* ＞＞ 64.233.162.8x

**Comment by weidongjian** [comment]
+1，厦门电信也挂了

**Comment by K-Guan** [comment]
@aofall ^。暂时不方便测试，各位帮忙测试下。如果没问题的话我就关issue了。

**Comment by 4679** [comment]
@K-Guan 
湖南电信测试无效

**Comment by zhao94254** [comment]
山西移动 谷歌hk能用

**Comment by nxtreaming** [comment]
如果还不能用，请邮件我（通过github上的我的profile可以找到邮件地址），我给大家提供一个免费代理，速度不快，但保证稳定性。

**Comment by K-Guan** [comment]
> 经路由追踪发现Google 主动屏蔽大陆的电信联通用户 :(
> 移动铁通教育网没事

@aofall 看来……没变化（参考@4579和@zhao94254的评论）

**Comment by K-Guan** [comment]
@nxtreaming 虽然知道你是好心，也无意冒犯。但是怎么总感觉是在打广告似的？ :)

**Comment by 4679** [comment]
google为何要这样做(;´Д`)

**Comment by nxtreaming** [comment]
免费的，并且我也没有把我邮件地址直接写出来，你要好是觉得是广告那也没办法了。

主要内部团队（10）使用（主要是代理我们的subversion和git访问，我们代码托管在国外某代码托管网站，不知道什么原因导致中国电信网络环境下访问慢），
10M带宽，主要用来使用google服务，并加速github等开发者网站访问。

On Thu, Jan 28, 2016 at 5:35 PM, Kevin Guan <notifications@github.com>
wrote:

> @nxtreaming <https://github.com/nxtreaming>
> 虽然知道你是好心，也无意冒犯。但是怎么总感觉是在打广告似的？ :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/racaljk/hosts/issues/181#issuecomment-176084255>.
>



-- 
Founder of http://coogle.io

**Comment by Eamonnzhang** [comment]
http://laod.cn/hosts

**Comment by K-Guan** [comment]
@nxtraming 嗯，我开个玩笑而已。

**Comment by K-Guan** [comment]
@4679 I have no idea.

**Comment by 4679** [comment]
惨案(´_ゝ`)

**Comment by aofall** [comment]
@K-Guan .81是不能用的啊……不提供正常的Web服务
82 83 84 ping之后选一个

**Comment by K-Guan** [comment]
@aofall （你说清楚呀，我看了下发现我们的hosts中有个81……）

**Comment by xuexin** [comment]
更新了最新的host,可以使用google.com.hk
不能使用google.com

**Comment by luke2008** [comment]
湖南联通表示还是不行，能ping通，但是无法访问

**Comment by racaljk** [comment]
爪机无能为力...

**Comment by p0mp0k0** [comment]
gmail可以了，google.com也可以了，但developer.android.com还不行

**Comment by imcaffrey** [comment]
@Eamonnzhang laod的hosts好多来自这个项目

**Comment by yyan** [comment]
developer.android.com 那一行的ip没有更新，修改成 64.233.162.82就可以了

**Comment by redapple0204** [comment]
@K-Guan 为什么现在还不能fq啊

**Comment by K-Guan** [comment]
好吧……我就是忙了一天而已……

@yyan 感谢： #e410168 
@redapple0204 怎么？还是不行吗？

**Comment by chy9002** [comment]
还有部分IP是64.233.160.90，这部分也ping不通了。

**Comment by redapple0204** [comment]
K-Guan 可以了，谢谢（其实就是垃圾杀软拦截）

**Comment by redapple0204** [comment]
@K-Guan  不过访问速度很慢，有时会出现无法访问，facebook也挂了。

**Comment by K-Guan** [comment]
@redapple0204 这事情别找我，我只是“奉命行事”。

**Comment by K-Guan** [comment]
@chy9002 感谢提醒，正在解决。

**Comment by K-Guan** [comment]
既然问题解决（至少，Google能访问了），我就先关闭此issue。如有其他问题，可以在这里ping我或另开一个issue。

**Comment by wolfan** [comment]
报告湖南联通google相差的服务不是很稳定哦，时不时不能用。
